DoSomething/Data
# new accounts from join now analysis

#### What's this PR do?
Adds files for estimating the number of new accounts created from adding the Join Now button in various locations and sending an email with the updated weekly counts.

#### How should this be manually tested?
Run join_now_accounts.py to confirm that an email is sent to/from your email address. Also note that you will need to create an env variable called `EMAIL_PW` and setup access for gmail 

#### Any background context you want to provide?
This was part of an effort in Q4 to understand the impact of adding the ability to create an account in new locations. As a result, it's unlikely to be used but this is meant to serve as a backup in case!
